_health-care/_js/_form.js

```diff
@@ -18,17 +18,60 @@ var HealthApp = (function() {
     var dateString = "";
 
     if (date.getMonth() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += (date.getMonth() + 1) + "/";
 
     if (date.getDate() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += date.getDate() + "/";
 
     return dateString + date.getFullYear();
   }
+  function extractMonetaryValueTransform(formData, arg) {
+    return Number(extractTransform(formData, arg)).toFixed(2).toString();
+  }
+  function validatePresence(input) {
+    var valid = true;
+
+    if (input.value === '') {
+      valid = false;
+    }
+
+    return valid;
+  }
+  function validateMonetaryValue(input) {
+    var valid = true;
+    var amount = input.value;
+
+    if (/[$a-zA-Z,]+/.test(amount)) {
```

Comment:
I was thinking about the problems with specifying what's not allowed (can't possibly assume everything the user might input), so I like this a lot better. Thanks!